core/chains/evm/txmgr/orm.go

```diff
@@ -177,34 +209,52 @@ func (o *orm) PreloadEthTxes(attempts []EthTxAttempt) error {
 
 // EthTransactions returns all eth transactions without loaded relations
 // limited by passed parameters.
-func (o *orm) EthTransactions(offset, limit int) (txs []EthTx, count int, err error) {
-	sql := `SELECT count(*) FROM eth_txes WHERE id IN (SELECT DISTINCT eth_tx_id FROM eth_tx_attempts)`
-	if err = o.q.Get(&count, sql); err != nil {
+func (o *orm) EthTransactions(offset, limit int, opts ...any) (txs []EthTx, count int, err error) {
+	qopts, err := ToQOpts(opts)
+	if err != nil {
+		err = errors.Wrap(err, "EthTransactions failed")
 		return
 	}
+	qq := o.q.WithOpts(qopts...)
```

Comment:
perhaps create a function to get qq from qopts, since you do that in many places.
getQueryer(opts ...any) (Q, error)

Just saves 1 line, but in many places.